src/pages/Chat/steps/LivingSituationInput.test.tsx

```diff
@@ -4,6 +4,7 @@ import * as React from 'react'
 import { createSession } from 'utils/sessionStorage'
 import { MockStorage } from 'utils/storage/MockStorage'
 import { WithStorageProps } from '../../../App'
+import { InputValidationError } from '../../../components/userInput/UserResponse'
```

Comment:
hehe can we fix the ../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../../? 😄

Reply:
sure ../../../../../../

Reply:
👉 `babel-plugin-module-resolver`

Reply:
Webpack and jest does it for us already :)